src/main/groovy/org/transmartproject/batch/patient/DemographicVariable.groovy

```diff
@@ -42,7 +39,6 @@ enum DemographicVariable {
     )
 
     String column
-    Object defaultValue
```

Comment:
Behavior of what? It does change the contract. We stopped to insert `0` as default value for age and `Unknown` as default value for gender.